src/components/ImportParameters.jsx

```diff
@@ -0,0 +1,158 @@
+import { useState } from "react";
+
+const parameterDetails = {
+  resource: { 
+    name: <><code>resource</code></>, 
+    type: "IRI, String", 
+    description: <>The file name to write to. 
+      If it contains an extension, this is used to automatically set the <code>compression</code> parameter. 
+      If set to the empty string <code>""</code>, the tuples are read from stdin until pressing <code>Ctrl + D</code>. 
+      This is restricted to one stdin resource per program. If omitted, this is set based on the predicate name, 
+      file format and compression type: <code>&lt;predicate-name&gt;.&lt;format&gt;.&lt;compression&gt;</code>.</> 
+  },
+  format: { 
+    name: <><code>format</code></>, 
+    type: "tuple, single value", 
```

Comment:
`single value` is not a type (neither is tuple, rather, it's a tuple of Nemo type names).